docs/src/app/components/pages/components/TimePicker/ExampleComplex.jsx

```diff
@@ -3,13 +3,17 @@ import TimePicker from 'material-ui/lib/time-picker/time-picker';
 
 export default class TimePickerExampleComplex extends React.Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {value24: null, value12: null};
```

Comment:
Can someone explain this behavior?
![fevr 12 2016 13 08](https://cloud.githubusercontent.com/assets/3165635/13006534/cbdb327a-d189-11e5-834a-9428f752406b.gif)
`value24` and `value12` are supposed to be decoupled :flushed:.